packages/lesswrong/components/questions/RelatedQuestionsList.tsx

```diff
@@ -41,6 +41,8 @@ const RelatedQuestionsList = ({ post, classes }: {
   
   const showParentLabel = sourcePostRelations.length > 0
   const showSubQuestionLabel = (sourcePostRelations.length > 0) && (targetPostRelations.length > 0)
+  
+  if (!totalRelatedQuestionCount) return null
```

Comment:
This component was adding a bunch of extra margin, and I think "related questions" are used ~never, so I figured this was fine.